custom-vu/src/main/kotlin/jces1209/vu/page/CloudIssuePage.kt

```diff
@@ -60,6 +67,116 @@ class CloudIssuePage(
         return CloudIssueLinking(driver)
     }
 
+    override fun isTimeSpentFormAppeared(): Boolean {
+        try {
+            driver
```

Comment:
What about jut to use ExpectedConditions.or()? so we should not wait until timeout exception